### PR TITLE
fix: convert token to usd in airdrop command

### PR DESCRIPTION
**Pull request by quanghuynguyen1902**

### CODE REVIEW

**Review by haongo138 — APPROVED** [review]

File: src/commands/airdrop/index/processor.ts [hunk 1]
```diff
@@ -262,7 +263,7 @@ export async function handleAirdrop(
 ) {
   const payload = await processor.getAirdropPayload(msgOrInteraction, args)
   const { amount, token, all } = payload
-  const { balance, usdBalance } = await validateBalance({
+  const { balance } = await validateBalance({
```
Comment (trkhoi) [inline src/commands/airdrop/index/processor.ts:266]: why not use usdBalance

File: src/commands/airdrop/index/processor.ts [hunk 2]
```diff
@@ -271,11 +272,16 @@ export async function handleAirdrop(
   if (all) payload.amount = balance
 
   const tokenEmoji = getEmoji(payload.token)
-  const usdAmount = (usdBalance / balance) * payload.amount
+  let usdAmount = await convertToUsdValue(payload.amount, payload.token)
```
Comment (trkhoi) [inline src/commands/airdrop/index/processor.ts:275]: pls provide screenshot for this bug fix
i dont want to repeat myself every time

**Review by trkhoi — CHANGES_REQUESTED** [review]

File: src/commands/airdrop/index/processor.ts (hunk 2 above)
Comment (quanghuynguyen1902) [reply src/commands/airdrop/index/processor.ts:275]: Screen Shot 2023-04-04 at 2 12 37 PM

File: src/commands/airdrop/index/processor.ts (hunk 1 above)
Comment (quanghuynguyen1902) [reply src/commands/airdrop/index/processor.ts:266]: It return 0
Screen Shot 2023-04-04 at 2 14 11 PM

File: src/commands/airdrop/index/processor.ts (hunk 2 above)
Comment (quanghuynguyen1902) [reply src/commands/airdrop/index/processor.ts:275]: Screen Shot 2023-04-04 at 2 15 56 PM

File: src/commands/airdrop/index/processor.ts (hunk 1 above)
Comment (trkhoi) [reply src/commands/airdrop/index/processor.ts:266]: need to check the logic in validateBalance to see why it return 0

**Comment by quanghuynguyen1902** [comment]
Screen Shot 2023-04-04 at 10 17 36 PM

**Review by trkhoi — APPROVED** [review]

**Pull request merged**